client/src/pages/ProfileListing/CoffeCard.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import Card from '@material-ui/core/Card';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import Typography from '@material-ui/core/Typography';
+import { IconButton, CardMedia } from '@material-ui/core';
+import LocationOnIcon from '@mui/icons-material/LocationOn';
+import Rating from '@mui/material/Rating';
+
+const CoffeCard = (props) => {

```

Comment:
Not sure what `CoffeCard` means, could we make sure we are doing better naming of our components.